src/svgstore.js

```diff
@@ -53,12 +53,19 @@ function svgstore(options) {
 					var oldDefId = elem.attr('id');
 					var newDefId = id + '_' + oldDefId;
 					elem.attr('id', newDefId);
+
+					/* process use tags */
 					child('use').each(function (i, use) {
 						if (child(use).prop('xlink:href') !== '#' + oldDefId) {
 							return;
 						}
 						child(use).attr('xlink:href', '#' + newDefId);
 					});
+
+					/* process fill attributes */
+					child('[fill="url(#' + oldDefId + ')"]').each(function (i, use) {
```

Comment:
I think we might still be missing some use cases here. I think `url(#)` references are valid in more than just `xlink:href` and `fill` attributes. I'm trying to track down a list of attributes where `url()` is supported.

Reply:
Yeah, there might be such cases, but I haven't found any other examples in our SVG exported from Sketch (about 100 icons). Please, let me know if you know such cases, otherwise, I'd suggest releasing this as it is now since it's under a feature flag, so it won't break anything for current users.